StRoot/StFcsDbMaker/StFcsDb.h

```diff
@@ -256,6 +257,13 @@ class StFcsDb : public TDataSet {
   void setPedestal(int ehp, int ns, int dep, int ch, float ped); //! setting pedestal
   void readPedFromText(const char* file="fcsped.txt"); //! reading pedestal from text
   
+  //g2t track info
+  unsigned int backTraceG2tTrack(unsigned int id, g2t_track_st* g2ttrk);
+  g2t_track_st* getParentG2tTrack(StFcsHit* h,      g2t_track_st* g2ttrk, float& fraction, unsigned int order=0);
+  g2t_track_st* getParentG2tTrack(StFcsCluster* c,  g2t_track_st* g2ttrk, float& fraction, unsigned int order=0);
+  g2t_track_st* getPrimaryG2tTrack(StFcsHit* h,     g2t_track_st* g2ttrk, float& fraction, unsigned int order=0);
+  g2t_track_st* getPrimaryG2tTrack(StFcsCluster* c, g2t_track_st* g2ttrk, float& fraction, unsigned int order=0);
```

Comment:
You should use `std::tuple` to return multiple values. Passing inputs as function arguments is more natural and is inline with standard math notation.
Please use `const` for immutable arguments. These are public methods, so you need to explain the meaning of the arguments with doxygen comments.

Reply:
At
  /star/u/akio/fcsmc/star-sw/StRoot/StFcsDbMaker/StFcsDb.h (and .cxx)
I'm having issue with CINT:

.sl73_gcc485/obj/StRoot/StFcsDbMaker/StFcsDbMaker_Cint.cxx: In function 'int G__StFcsDbMaker_Cint_646_0_93(G__value*, const char*, G__param*, int)':
.sl73_gcc485/obj/StRoot/StFcsDbMaker/StFcsDbMaker_Cint.cxx:1338:73: error: invalid use of 'class std::tuple<g2t_track_st*, float, int>'
       G__letint(result7, 105, (long) ((StFcsDb*) G__getstructoffset())->std::tuple<g2t_track_st*,float,int> getParentG2tTrack((StFcsHit*) G__int(libp->para[0]), (g2t_track_st*) G__int(libp->para[1])
                                                                         ^
.sl73_gcc485/obj/StRoot/StFcsDbMaker/StFcsDbMaker_Cint.cxx:1342:73: error: invalid use of 'class std::tuple<g2t_track_st*, float, int>'
       G__letint(result7, 105, (long) ((StFcsDb*) G__getstructoffset())->std::tuple<g2t_track_st*,float,int> getParentG2tTrack((StFcsHit*) G__int(libp->para[0]), (g2t_track_st*) G__int(libp->para[1])));

Reply:
Ah, yes. I forgot about our ROOT5/rootcint not being c++11 compatible. Scratch the comment about the `std::tuple`...